apps/ios/src/Podfile

```diff
@@ -28,6 +28,8 @@ use_test_app! do |target|
     pod 'FluentUI-React-Native-Avatar', :path => '../../../packages/experimental/Avatar/FluentUIReactNativeAvatar.podspec'
     pod 'FluentUI-React-Native-Shimmer', :path => '../../../packages/components/Shimmer/FluentUIReactNativeShimmer.podspec'
     pod 'FluentUI-React-Native-Button', :path => '../../../packages/experimental/NativeButton/FluentUIReactNativeButton.podspec'
+    pod 'react-native-menu', :path => '../../../node_modules/@react-native-menu/menu'
+    pod 'RNCPicker', :path => '../../../node_modules/@react-native-picker/picker'
```

Comment:
lets either be generous and fix the personcoin or we don't feel lucky and remove this dependency